token/services/db/sql/common/wallet.go

```diff
@@ -44,7 +45,7 @@ func (db *WalletStore) CreateSchema() error {
 	return common.InitSchema(db.writeDB, []string{db.GetSchema()}...)
 }
 
-func (db *WalletStore) GetWalletID(identity token.Identity, roleID int) (driver.WalletID, error) {
+func (db *WalletStore) GetWalletID(ctx context.Context, identity token.Identity, roleID int) (driver.WalletID, error) {
 	idHash := identity.UniqueID()
 	result, err := common.QueryUnique[driver.WalletID](db.readDB,
```

Comment:
Since you are adding the context in the stores, you can also use the context and change these methods
```
QueryUnique -> QueryUniqueContext
Query -> QueryContext
Exec -> ExecContext
```